src/components/errorPage/404.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const page404 = () => (
+  <div className="row justify-content-center">
+    <div className="col-md-12 text-center">
+      <span className="display-1 d-block">404</span>
```

Comment:
You can use `div` to get rid of `d-block` here.

Reply:
Resolved.